app/controllers/iiif/auth/v2/probe_service_controller.rb

```diff
@@ -80,12 +80,21 @@ def add_detail(file)
           elsif file.embargoed?
             detail[:heading] = { en: [I18n.t('probe_service.embargoed', date: file.embargo_release_date.to_date)] }
           elsif file.restricted_by_location?
-            detail[:heading] = { en: [I18n.t('probe_service.location', location: Settings.user.locations.labels.send(file.location))] }
+            detail[:status] = 403
+            detail[:heading] = { en: [heading_html('location', location: Settings.user.locations.labels.send(file.location))] }
+          elsif file.no_download?
+            detail[:status] = 403
+            detail[:heading] = { en: [heading_html('no_download')] }
           end
           detail[:note] = { en: [I18n.t('probe_service.access_restricted')] }
           detail
         end
-        # rubocop:enable Metrics/AbcSize
+
+        def heading_html(heading_field, location: nil)
+          "#{I18n.t("probe_service.#{heading_field}_icon")}<p class='loginMessage'>#{I18n.t("probe_service.#{heading_field}",
```

Comment:
Can we leave the markup out of this PR?  This is going to couple the CSS across two different projects.  In my experience that gets' hard to follow, especially when we're not actively paying attention to both codebases, or a developer who is less familiar with this gets assigned to work on this.

Reply:
Done. Moved svg into icons. This makes a much cleaner code all around